config.py

```diff
@@ -66,6 +68,12 @@ def get_credentials():
     return CONFIG['github_user'], CONFIG['github_pw']
 
 
+def get_proxy():
+    if CONFIG['github_proxy'] == '':
```

Comment:
Whats the reason for the if check? `os.environ.get('GITHUB_PROXY', None)` will set `config['github-proxy']` to `None` if `GITHUB_PROXY` is not specified

Reply:
i.e. you can just return CONFIG['github_proxy']

Reply:
> Whats the reason for the if check? os.environ.get('GITHUB_PROXY', None) will set config['github-proxy'] to None if GITHUB_PROXY is not specified

But the configuration file will, I think, return an empty string if the proxy is not specified. Well, I am going based off of the `get_credentials()` function. 

```
  config['github_proxy'] = conf.get('GitHub', 'proxy')
```

Reply:
Updated the `conf.get` to set a fallback value to `None`. Now, it is returning `CONFIG['github_proxy']`